packages/medusa/src/services/totals.ts

```diff
@@ -906,6 +931,13 @@ class TotalsService extends TransactionBaseService {
       }
 
       lineItemTotals.total += lineItemTotals.tax_total
+
```

Comment:
todo: I think there has been a problem during your merge. it should not be that way

Reply:
just to be sure could you check that there is no other un expected merge problem please :)